content/en/dashboards/change_overlays/_index.md

```diff
@@ -32,7 +32,7 @@ To get started, click **Show Overlays** in the upper right corner of your dashbo
 
 {{< img src="dashboards/change_overlays/show_overlays_button.png" alt="Overlays button on dashboard header" style="width:100%;">}}
 
-When actived, the **Service** search bar will display the **Most Relevant** service by default. Datadog automatically selects the service most frequently referenced in the queries supporting the widgets on the dashboard.
+When activated, the **Service** search bar will display the **Most Relevant** service by default. Datadog automatically selects the service most frequently referenced in the queries supporting the widgets on the dashboard.
```

Comment:
```suggestion
When activated, the **Service** search bar displays the **Most Relevant** service by default. Datadog automatically selects the service most frequently referenced in the queries supporting the widgets on the dashboard.
```
While we're touching this line, let's make its verb tense conform to the style guide.